openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -62,7 +62,7 @@ def get_account_settings(request, username=None, configuration=None, view=None):
             "shared", only shared account information will be returned, regardless of `request.user`.
 
     Returns:
-         A dict containing account fields.
+         A dict containing account fields if only one user is requested else returns a list of dict
```

Comment:
It looks like we already have a distinction between the two endpoints.  We just need to use it.  See below.